src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2608,6 +2608,9 @@
   <data name="InvalidOperation_HandleIsNotPinned" xml:space="preserve">
     <value>Handle is not pinned.</value>
   </data>
+  <data name="InvalidOperation_InvalidUtf8" xml:space="preserve">
+    <value>Formatted string contains characters not representable as valid UTF8.</value>
```

Comment:
we are not consistently using either UTF8 or UTF-8 in our strings. I see 10 of one and 15 of the other.

It appears the correct spelling is UTF-8 per [ Unicode® Technical Report #17
UNICODE CHARACTER ENCODING MODEL](https://www.unicode.org/reports/tr17/)